activities/communication/communications-plan.md

```diff
@@ -0,0 +1,169 @@
+
+## Calls to action
+
+### Decision makers and sponsors
+
+This includes:
+
+* [political](../value-and-impact/user-mapping/political.html)
+* [senior civil servants](../value-and-impact/user-mapping/senior-civil-servants.html)
+* [structural funding and philanthropy](../value-and-impact/user-mapping/structural-funding-philanthropy.html)
+
+Messages:
+
+* Find out more about [place or institution’s] interesting public code(ish) approach/project
+* Find out more about the Foundation for Public Code and its work (new member signed up, new codebase incubated)
+* Find out more about the Foundation for Public Code its vision
+* International movements/partnerships for being smarter about technology/digital infrastructure (“you want to be part of this club of innovators”)
+* Find out more about the Standard and certification (how we contribute to risk reduction)
+* Find out more about codebase reuse (multiplying tech investment or saving money)
+
+### Users and contributors
+
+This includes:
+
+* the policy community
+* [the tech and open source communities](../value-and-impact/user-mapping/developers-and-designers.html)
+* [business management and procurement for public administrations](../value-and-impact/user-mapping/management-and-procurement.html)
+
+Messages:
+
+* Find out more about [place or institution’s] interesting technological approach to public code or open source (objective: creating an enabling environment within government by showing this is normal elsewhere)
+* Find out more about [place or institution’s] interesting policy approach to public code or open policymaking (objective: creating an enabling environment within government by showing this is normal elsewhere)
+* Find out more about [place or institution’s] interesting management approach to procuring or producing public code or open source code (objective: creating an enabling environment within government by showing this is normal elsewhere)
+* Find out more about the Standard and our other resources for different user groups
+* Come to meetups and events
+* News from and profiles of our contributor community (‘join our thriving community!’)
+
+### Other interested parties
+
+This includes:
+
+* vendors and the private sector
+* end users (residents and citizens)
+* journalists, academia and non-governmental organizations
+
+Messages:
+
+* Good news from our partners and affiliates (eg, relevant session at organization X’s annual conference!)
+* Any news about Foundation or codebase impact for end users (eg, check out these communications by a member about their new software for managing parking)
+* State of the market info about/for vendors (focusing on a class of vendors, rather than endorsing specific vendors)
+* Potentially: announcing when a new vendor becomes available to support a specific codebase (like [this OS2 example](https://os2.eu/blog/en-ny-leverandoer-til-os2kitos-fra-juni-2019))
+
+## Channels
+
+Our channels are:
```

Comment:
Should we consider adding events as a channel? They are mentioned later on as things we do campaigns on, but are not explained anywhere here.

This would make our channels of communication more diverse, and not just restricted to social media platforms + blog. 

A thought for later: when we (re)establish our press contacts, would these also be worth mentioning here?

Reply:
> [This comment has a soundtrack.](https://www.youtube.com/watch?v=WySzEXKUSZw)

I don't consider events a communications channel. The other channels are persistent, free to access, can be indexed by search engines and found well into the future, and all contain (interlinked or cumulative) content that we create for others to consume privately and asynchonously. Though we often communicate at events and host our own, and craft original content for them, most of the magic comes from being in the room with the other participants. 
It's best practice for us to share our event content publicly afterwards so it's accessible to people who couldn't make the event, but we do that  via the other communcations channels.